### DISCUSSION TITLE
Over pressurizing with the oxygen generator to return from the abbys

### DISCUSSION THREAD

**Discussion by Zimon3283** [FakeFishGames/Barotrauma#14431] [Ideas and suggestions]
How about to be able to dive deeper or come back from the abbys you higher the inside air pressure with the oxygen generator? If your already breached you need to time it well with the pumps. And filling your submarine with more oxygen with a machine that isn't designed to do that makes fires more probable and dangerous, sucks a bunch of power, and breaks said machine. Of course higher pressure might need diving suits and if you have a single holde all the pressure is gone quite quickly so don't bump rocks.

**Comment by Zimon3283** [comment]
Of course it needs a signal in for wiering my beloved

**Comment by Zimon3283** [comment]
It could also just drain a bit of power to keep the pressure stable and as soon as there is a minus in pressure it needs a lot again and then suddenly stops needing a lot again to break the junktions that already have a hightened fire risk

**Comment by Next-Door-Tech** [comment]
Hard "no" from me on this one. From a real-world physics standpoint, this wouldn't work for a multitude of reasons - increased fire risk being the least of these.

---

In very short and simple terms, in order for a submarine be positively buoyant (i.e. want to float upwards on its own) the average density of the sub and everything inside needs to be less than that of the water surrounding it. In other words, you need to make the sub *lighter* to make it go up. Normally you achieve this by pumping water out of the ballast tanks to reduce the total mass of the submarine, while keeping the amount of air inside the same.

Now, let's assume our submarine is neutrally buoyant (floating in place) and we try increasing the air pressure inside the sub (without adjusting ballast). To do that, we have to pump _more_ air molecules into the submarine's interior. Air molecules have mass, meaning that if we pump more of them into the submarine, the total mass of the submarine *increases*. In other words, the sub becomes *heavier* and starts to **sink**, not float.

---

On top of that, where would all of this extra air come from? We're under a very thick layer of ice on Europa, so we can't exactly send up a snorkel to the surface, not to mention that there's no atmosphere above the ice anyway. (Technically there is, but it's one trillion times thinner than Earth's atmosphere.)

It *is* possible to turn water directly into hydrogen and oxygen gas using electrolysis, but then you have to choose between a pure hydrogen atmosphere (unbreathable) or pure oxygen atmosphere (extreme fire hazard, and actually toxic to humans at moderately elevated pressures) inside the submarine. Any amount of mixing the two together and the entire submarine would literally explode the instant one of those electrical boxes sparks.

Having an inert gas in your breathing air is actually incredibly important, so I presume nitrogen is actually a valuable commodity under the seas of Europa.

**Reply by Zimon3283** [reply]
First of all I do NOT intend to make it lighter (why does everyone say that? Is my English this horrible?) I want to counter the outside pressure to keep the hull intactSecond the toxicity and flammability of the atmosphere is meant so players don't always use it and only if they really need it.

**Reply by Zimon3283** [reply]
Ups noticed I did not say I want to counter outside pressure sorry

**Comment by ThatGuyCalledBlank** [comment]
Here's an idea: Inflatable portions of the submarines? Very weak, but useful. Like an extra ballast but temporary? Would make some funny workshop additions to the community submarines page.

**Reply by ThatGuyCalledBlank** [reply]
Or EVEN: It just being a goofy alternative to normal submarines. Pump air in and out instead of water with the caviat of thinning the submarine's own atmosphere.

**Comment by GuzevArtem** [comment]
I'd like the other way around.
High-pressure pumps to allow the creation of high-pressure ballast rooms. So the same room volume will provide more 'ballast' force, but will cause flooding if any side is breached/opened.
In theory could be done and the other way around - by pumping out the atmosphere, it will create effectively vacuum and allow more lift from the same volume as well.
